hedera-mirror-rosetta/build/supervisord.conf

```
@@ -10,7 +10,7 @@ nodaemon=true               ; start in foreground if true; default false
 file=/tmp/supervisor.sock
 
 [program:postgres]
-command=/usr/lib/postgresql/9.6/bin/postgres -c config_file=/etc/postgresql/9.6/main/postgresql.conf
+command=/usr/lib/postgresql/%(ENV_PG_VERSION)s/bin/postgres -c config_file=%(ENV_PGCONF)s/postgresql.conf
```

You shouldn't need to use an absolute path since you put this location on the $PATH. And the postgresql.conf should get read automatically if it's placed within the data directory. Maybe try putting it there in the standard location.

```suggestion
command=postgres
```

---

I meant instead of putting it in etc you can just put it data dir and it should automatically be picked up, avoiding manual configuration.

---

moved the db configuration files to `/var/lib/postgresql/13/main` and symlink it back to where `/etc/init.d/postgresql` wants it to be.

now the command in supervisord.conf is simplified to just `postgres`.

p.s. $PATH set in the Dockerfile isn't available to non-root users with su for instance `su postgres`, but it's available with supervisor running the process under `postgres` user